css/site.styl

```diff
@@ -107,13 +114,29 @@ img
     opacity: 1
   100% 
     opacity: 1
-  
-input
-  height: 0
-  width: 0
-  opacity: 0
-  text-transform: uppercase
 
+// CONVERSATION FORM
+.conversation
+
+  #custom-form
```

Comment:
how can an ID be nested under a class? There is only ever 1 `#custom-form` but more than one `.conversation`?